client/index.ts

```diff
@@ -33,6 +32,16 @@ interface AppointmentReceipt {
     readonly watcherAddress: string;
 }
 
+// Encode the topics in the format expected from Pisa's contract.
```

Comment:
This should be a private static function on the Client class

Reply:
Done in 17fdde9.